src/setup/components-global.js

```diff
@@ -48,13 +32,6 @@ export default {
       });
     } else {
```

Comment:
todo: the `else` can be removed. it was only added for PhpStorm so it knows, which components are already imported on global scale. Since we only import components that are most likely not imported by other components we can drop this.